docs/advanced_configuration.md

```diff
@@ -2,6 +2,35 @@
 
 If you have not set up the SDK yet, follow the [in-app setup instructions][1] or refer to the [React Native RUM setup documentation][2].
 
+## Testing with Jest
+
+Testing apps using `'@datadog/mobile-react-native'` may require some extra steps since Native Modules are non existent in testing environments.
```

Comment:
```suggestion
Testing apps using `'@datadog/mobile-react-native'` might require completing extra steps, since Native Modules do not exist in testing environments.
```